PKGBUILD

```diff
@@ -37,15 +37,36 @@ build() {
   cd "$srcdir/atom-$pkgver"
 
   export PYTHON=python2
-  script/build --build-dir "$srcdir/atom-build"
+  script/build --compress-artifacts
```

Comment:
You're correct; I was thinking this flag might have some impact on the compiled `atom` binary size (it does not, in fact, have any such effect).